code/datums/wires/wires.dm

```diff
@@ -1,119 +1,37 @@
-var/list/wire_colors = list( // http://www.crockford.com/wrrrld/color.html
-	"aliceblue",
-	"antiquewhite",
+var/list/wire_colors = list(
 	"aqua",
-	"aquamarine",

```

Comment:
this is a legitimate colour, just as valid as some of the others you've left intact.